From: Tommy Webb <tommy@calyxinstitute.org>
Date: Tue, 27 May 2025 16:06:23 +0000
Subject: Hide internally-inactive leak detection preference

Leak detection is disabled internally by:
Disable-privacy-issues-in-password-manager.patch

So, hide the preference that controls it.

Change-Id: I40d58e8e34fd92c4dc26c5dec7dd5d1ab0394041
---
 chrome/android/java/res/xml/privacy_preferences.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/android/java/res/xml/privacy_preferences.xml b/chrome/android/java/res/xml/privacy_preferences.xml
--- a/chrome/android/java/res/xml/privacy_preferences.xml
+++ b/chrome/android/java/res/xml/privacy_preferences.xml
@@ -74,6 +74,7 @@ found in the LICENSE file.
         android:key="password_leak_detection"
         android:title="@string/passwords_leak_detection_switch_title"
         android:summary="@string/passwords_leak_detection_switch_summary"
+        app:isPreferenceVisible="false"
         android:persistent="false" />
     <!-- Old HTTPS-First Mode toggle. Hidden if HttpsFirstBalancedMode flag is enabled.
          TODO(crbug.com/349860796): Remove once new settings are fully rolled out. -->
-- 

